keyserver/coniksserver/coniksserver.go

```diff
@@ -0,0 +1,153 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/signal"
+	"path"
+	"strings"
+
+	"github.com/BurntSushi/toml"
+	"github.com/coniks-sys/coniks-go/crypto/sign"
+	"github.com/coniks-sys/coniks-go/crypto/vrf"
+	"github.com/coniks-sys/coniks-go/keyserver"
+	"github.com/coniks-sys/coniks-go/utils"
+)
+
+type boolStringFlag struct {
+	set   bool
+	value string
+}
+
+func (sf *boolStringFlag) Set(x string) error {
+	sf.set = true
+	if x != "true" {
+		sf.value = x
+	}
+	return nil
+}
+
+func (sf *boolStringFlag) String() string {
+	return sf.value
+}
+
+func (sf *boolStringFlag) IsBoolFlag() bool {
+	return true
+}
+
+func main() {
+	genkey := boolStringFlag{false, "sign,vrf"}
+	flag.Var(&genkey, "genkey", "Generate signing/VRF key, e.g. -genkey=sign,vrf")
+	genConfig := flag.Bool("genconfig", false, "Generate server configurations file")
+	dir := flag.String("dir", ".", "Directory location for storing generated files")
+	confFile := flag.String("config", "config.toml", "Path to the configuration file")
+
+	flag.Parse()
+
+	switch {
+	case *genConfig && genkey.set:
+		mkConfig(*dir)
+		mkKey(genkey.value, *dir)
+		return
+	case *genConfig:
+		mkConfig(*dir)
+		return
+	case genkey.set:
```

Comment:
`boolStringFlag` is a bool type flag, so its syntax is `-genkey=sign,vrf`. The reason is I want to have a call like `./coniksserver -genkey` with default value. I'm not sure if we want to change it to string type flag.